tests/integration/test_design_export.py

```diff
@@ -138,7 +138,11 @@ def test_export_to_parasolid_text(modeler: Modeler, tmp_path_factory: pytest.Tem
     # Define the location and expected file location
     location = tmp_path_factory.mktemp("test_export_to_parasolid_text")
 
-    if modeler.client.backend_type == BackendType.LINUX_SERVICE:
+    if modeler.client.backend_type in (
+        BackendType.LINUX_SERVICE,
+        BackendType.CORE_LINUX,
+        BackendType.CORE_WINDOWS,
+    ):
```

Comment:
Same as before - naming convention should only be for linux-based systems right?